gallery/src/components/NxTree/NxTreeNoGutterExample.tsx

```diff
@@ -105,6 +105,15 @@ export default function NxTreeCollapsibleExample() {
                                 <span>cucumber.webm</span>
                               </NxTree.ItemLabel>
                             </NxTree.Item>
+                            <NxTree.Item>
+                              <NxTree.ItemLabel>
+                                <NxTextLink external
+                                            className="nx-dropdown-link"
```

Comment:
This classname shouldn't be here.  This isn't a dropdown, and the styling effects that the class has are incorrect here.

Reply:
> This classname shouldn't be here. This isn't a dropdown, and the styling effects that the class has are incorrect here.

I copied the code from the dropdown page but forgot to remove the class, removed here: 0aac6314d6ad47a8895ab77c4b87d5b57339c786

Reply:
Updated visual tests: 69e9d24ed7701ed9fdcc973182e46474a1f64e2d